dlight/dlight.remote.ui/src/org/netbeans/modules/remote/api/ui/FileChooserUIImpl.java

```diff
@@ -3509,29 +3504,6 @@ private void enableAllButCancel(boolean enable) {
 
     private void changeDirectory(File dir) {
         JFileChooser fc = getFileChooser();
-        // Traverse shortcuts on Windows
-        if (dir != null && FilePane.usesShellFolder(fc)) {
```

Comment:
What happens on windows with the removal of this? Will the user have a bad experience or will it still work?

Reply:
Difficult to answer as the FilePane API is not well documented. I assume users will either complain or will be happy to run NetBeans on any Java 11 implementation.

We can either: 
- Use reflection to try to restore the previous behaviour.
- Investigate further in the logs why this was here in the first place.

Reply:
Tested with windows and it at least does not blow immediately. I suggest to comment the code out here and leave it in place. I know, that we have version control, but I can acknowledge for me, that commenting this code out is "to get this working at all", not "this is the right solution". But I can live either way.

Reply:
Found it, I think. It seems JFileChooser was not very good at dealing with Windows shortcuts (.lnk) in the JVM 1.3.0/1.4.2/5.0 releases https://bugs.openjdk.java.net/browse/JDK-4356160

Things have evolved since then and JFileChooser and Windows shortcuts work well together now (not always, though https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8214924).

Reply:
Just seen your comment! Let's leave it commented out with a small description of what's going on. Let's wait for the current Travis build to finish, though.

Reply:
D'oh. 

It seems this very exact piece of NetBeans code (that we've deleted) ended up landing in the OpenJDK implementation of "BasicFileChooserUI.java"  here 
https://github.com/openjdk/jdk/blob/0a65e8b282fd41e57108422fbd140527d9697efd/src/java.desktop/share/classes/javax/swing/plaf/basic/BasicFileChooserUI.java#L1363 

So, AFAIU, there's no real need to implement it here (as we're extending JDK's BasicFileChooserUI). In fact I do prefer this code being mantained by the JDK Team, and not here in NetBeans.

Reply:
There we go with a commit with comments.